eventstore/tests/test_check.py

```diff
@@ -0,0 +1,102 @@
+# (C) Calastone Ltd. 2018
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+# System
+import subprocess
+import os
+import time
+
+# 3rd Party
+import pytest
+
+# Project
+from datadog_checks.eventstore import EventStoreCheck
+from datadog_checks.errors import CheckException
+from datadog_checks.utils.common import get_docker_hostname
+
+
+def test_config():
+    c = EventStoreCheck('eventstore', {}, {}, None)
+
+    # empty instance
+    instance = {}
+    with pytest.raises(CheckException):
+        c.check(instance)
+
+    # Timeout
+    instance = {'url': 'http://foobar'}
+    with pytest.raises(CheckException):
+        c.check(instance)
+
+    # Statuscode
+    instance = {'url': 'http://google.com'}
+    with pytest.raises(CheckException):
+        c.check(instance)
+
+    # Decode Error
+    instance = {'url': 'https://google.com'}
+    with pytest.raises(CheckException):
+        c.check(instance)
+
+
+@pytest.fixture
+def aggregator():
```

Comment:
since this was already defined in `conftest.py`, pytest will make it automatically available to all the test modules, you can remove the function from here.